<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessITKQtVersionClass</class>
 <widget class="QMainWindow" name="ImageProcessITKQtVersionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>595</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>859</width>
    <height>595</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>859</width>
    <height>595</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NewImageProcessProgram</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ImagePseudoColor.ico</normaloff>ImagePseudoColor.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_fusion">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_white">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>Photograph</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>171</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>High</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_highValue">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>65535</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_background">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Low</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_substract">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Substract
Background</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_lowValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65534</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_denoise">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Denoise</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_filterSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_lowValue">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>19</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65534</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_highValue">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>19</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>65535</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_photograph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Photograph</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBox_saveAll">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save all</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_luminescence">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Luminescence</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_luminescence">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>Luminescence</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_resize">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize to Luminescence</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_colormap">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>39</y>
      <width>41</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>JetColormapFunction.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_colormapHigh">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>65535</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_colormapLow">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>430</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_messageBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>841</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imageprocessitkqtversion.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>verticalSlider_lowValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_lowValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_highValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_highValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_lowValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_lowValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_highValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_highValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_lowValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_colormapLow</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>823</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_highValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_colormapHigh</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>839</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
